src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Figure[] arrayOfFigure = new Figure[6];
+        final int First_Half_Of_Array = 3;
+
+        for (int i = 0; i < First_Half_Of_Array; i++) {
+            arrayOfFigure[i] = FigureSupplier.getRandomFigure();
+        }
+        for (int i = First_Half_Of_Array; i < arrayOfFigure.length; i++) {
+            arrayOfFigure[i] = FigureSupplier.getDefaultFigure();
+        }
+        for (int i = 0; i < arrayOfFigure.length; i++) {
+            arrayOfFigure[i].drawFigure();
+            if (i >= First_Half_Of_Array) {
+                System.out.println();
+            }
```

Comment:
The check 'if (i >= First_Half_Of_Array)' and the corresponding 'System.out.println()' seem unnecessary. If you want to add a newline after drawing each figure, it should be inside the 'draw()' method of each figure class, not in the loop here.